### ISSUE TITLE
Add parameter to specify the proto

### ISSUE THREAD

**Issue by albertoperdomo** [nicolewhite/cycli#12]
This is in order to support instances secured with HTTPS.

**Comment by nicolewhite** [comment]
Any ideas how to do this with py2neo? When I try:

```python
from py2neo import Graph
graph = Graph("https://localhost:7473/db/data/“)
```

I get:

```
py2neo.packages.httpstream.http.SocketError: Exec format error
```

**Comment by albertoperdomo** [comment]
Have you tried including the username and password?
From what I know it should work

```
from py2neo import Graph

graph = Graph("https://user:pass@domain.com:7374/db/data/")
```

Any ideas @nigelsmall ?

**Comment by nicolewhite** [comment]
Auth is disabled

**Comment by albertoperdomo** [comment]
What version of py2neo are you using?

**Comment by nicolewhite** [comment]
2.0.7

**Comment by albertoperdomo** [comment]
According to the docs, it should work.

**Comment by nicolewhite** [comment]
lol

**Comment by nicolewhite** [comment]
Will you install from [this branch](https://github.com/nicolewhite/cycli/tree/https) and let me know if it works for you?

**Comment by albertoperdomo** [comment]
Seems to connect at first, but then fails. Output:

```
/usr/local/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
 ______     __  __     ______     __         __
/\  ___\   /\ \_\ \   /\  ___\   /\ \       /\ \
\ \ \____  \ \____ \  \ \ \____  \ \ \____  \ \ \
 \ \_____\  \/\_____\  \ \_____\  \ \_____\  \ \_\
  \/_____/   \/_____/   \/_____/   \/_____/   \/_/

Version: 0.1.1
Bug reports: https://github.com/nicolewhite/cycli/issues

> MATCH (n) RETURN n LIMIT 1



Can't assign requested address
>

Expecting object: line 1 column 226 (char 225)
>
```

**Comment by nicolewhite** [comment]
This was with the `-s` flag?

**Comment by albertoperdomo** [comment]
```
$ cycli -h host.domain.com -s -P port -u username
```

**Comment by nicolewhite** [comment]
And you say you had this working with py2neo? Can you show me the code exactly?

**Comment by albertoperdomo** [comment]
Turns out I ran that test with an outdated version. The error message `Can't assign requested address` was due to a bug in the URLs returned by the transactional endpoint in Neo4j 2.1.6. This happens with py2neo, too, because it tries connecting to `localhost:0`.

I just ran some tests with py2neo and cycli against 2.2.3 and it works correctly!

So sorry about the confusion!! Thanks for doing this!
